.gitignore

```diff
@@ -1,8 +1,6 @@
 # data
 data/
 **.db
-# Nats credentials
-**.creds
```

Comment:
I think we should keep this as we want to avoid accidentally committing NATS Creds files

Reply:
Ah, I removed it because it's a duplicate. Exists in line 18